inspectit-oce-core/src/main/resources/config/fallback.yml

```diff
@@ -0,0 +1,8 @@
+#------------------------
```

Comment:
what was the reasoning for having the fallback config that has nothing active? i mean the other option I see is to have the default one on. But I guess I am overlooking something..